intro/Readme.md

```diff
@@ -1,28 +0,0 @@
-What are the three most essential skills that a beginner programmer needs to know?
```

Comment:
No, you deleted the file...  Which when we merge the PR will delete it from master...